components/prism-llvm.js

```diff
@@ -0,0 +1,12 @@
+(function(Prism) {
+	Prism.languages.llvm = {
+		'comment': /;.*/,
+		'string': /"[^"]*"/,
+		'variable': /[%@!#](?:(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*|[0-9]+)/,
+		'label': /(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*:/,
+		'type': /\b(?:void|label|token|metadata|float|double|half)\b/,
+		'keyword': /[a-z_][a-z_0-9]*/,
+		'number': /[+-]?[0-9]+(?:.[0-9]+)?(?:[eE][+-]?[0-9]+)?|0x[0-9A-Fa-f]+/,
```

Comment:
Is `-0xABC` not a number?

Also, please add a test specifically for numeric literals like [this one](https://github.com/PrismJS/prism/blob/master/tests/languages/javascript/number_feature.test).

```suggestion
		'number': /[+-]?\b\d+(?:\.\d+)?(?:[eE][+-]?\d+)?\b|\b0x[\dA-Fa-f]+\b/,
```

Reply:
Interestingly -0xABC isn't a number because hexadecimal is only used for bit-exact floating point literals.